.github/workflows/db_import.yml

```diff
@@ -0,0 +1,65 @@
+name: DB Import
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
Maybe we also want to run when PR is targeting other branches? (useful when we try to stack a series of PRs on branches).

Reply:
No objections. I removed the branch filter.